ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -252,7 +255,7 @@ public void shouldBuildCorrectFormatsForV2NonWindowedTable() {
   public void shouldReplaceOldNonWindowedStreamSourceWithMatchingPseudoColumnVersion() {
     // Given:
     givenNonWindowedStream();
-    givenExistingQueryWithOldPseudoColumnVersion();
+    givenExistingQueryWithOldPseudoColumnVersion(windowedStreamSource);
```

Comment:
Why are we passing in a windowedStreamSource for a test that tests non-windowed stream sources?